content/v2.0/reference/flux/stdlib/experimental/http/get.md

```diff
@@ -0,0 +1,61 @@
+---
+title: http.get() function
+description: >
+  The `http.get()` function submits an HTTP GET request to the specified URL with headers
+  and returns the HTTP status code and body as a byte array.
+menu:
+  v2_0_ref:
+    name: http.get
+    parent: HTTP-exp
+weight: 301
+---
+
+The `http.get()` function submits an HTTP GET request to the specified URL with headers
+and returns the HTTP status code and body as a byte array.
+
+_**Function type:** Miscellaneous_
+
+{{% warn %}}
+The `http.get()` function is currently experimental and subject to change at any time.
+By using this function, you accept the [risks of experimental functions](/v2.0/reference/flux/stdlib/experimental/#use-experimental-functions-at-your-own-risk).
+{{% /warn %}}
+
+```js
+import "experimental/http"
+
+http.get(
+  url: "http://localhost:9999/",
+  headers: {x:"a", y:"b", z:"c"}
+)
+```
+
+## Parameters
+
+### url
+The URL to send the GET request to.
+
+_**Data type:** String_
+
+### headers
+Headers to include with the GET request.
+Headers are represented by key-value pairs.
+
+_**Data type:** Object_
+
+## Examples
+
+##### Get the status of InfluxDB
+```js
+import "influxdata/influxdb/secrets"
+import "experimental/http"
+
+token = secrets.get(key: "READONLY_TOKEN")
+
+response = http.get(
+    url: "http://localhost.com:9999/health",
+    headers: {Authorization: "Token ${token}"}
+  )
```

Comment:
Have you actually made post calls with the headers this way?  I'm no expert on calling/using headers and I'm not saying it does not work, but it doesn't look like my working use case.  Here's the example of the types of header use I was testing:

http.get(url: "https://free-nba.p.rapidapi.com/games/44", headers: {"x-rapidapi-host": "free-nba.p.rapidapi.com","x-rapidapi-key": "a64aa239d5..."})

Reply:
I haven't tested this function specifically (haven't built Flux from source), but it uses the same syntax as `http.post` and I have tested that. Keys inside of objects are always treated as strings, so you shouldn't have to explicitly wrap them in quotes.